test/ui/login-test.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/* Basic POC test to verify operation of installer login screen
+   when invalid username/password is entered
+   ldimaggi, March 2016
+   */
+
+let webdriver = browser.driver;
+let ACCOUNT_URL = '/account';
+let PAGE_TITLE = 'Red Hat Developer Platform Installer';
+let BAD_USERNAME = "badusername1";
+let BAD_PASSWORD = "badpassword1";
+var loginButton;
+
+describe('Login', function() {
+
+  beforeEach(function() {
+    loginButton = protractor.until.elementLocated(By.className('btn-lg'))
+  });
+
+/* Simple test for page URL */
+  it ('Inital page URL should be correct', function() {
+    expect (browser.getLocationAbsUrl()).toEqual (ACCOUNT_URL);
+  });
+
+/* Simple test for page title */
+  it ('Initial page title should match', function() {
+    expect (browser.getTitle()).toEqual(PAGE_TITLE);
+  });
+
+/* At this point, the login button should be disabled */
+  it ('Login button should be disabled until user enters name/password', function() {
+    webdriver.wait(loginButton, 10000)
+    .then(function(elm) {
+      expect(elm.isEnabled()).toEqual(false);
+    });
+  });
+
+describe('Bad username and password', function() {
+  beforeEach(function() {
+    var usernameField = element(by.id('username'));
+    var passwordField = element(by.id('password'));

```

Comment:
Only one retrieval of the elements is needed - would be better to move the element discovery to a local beforeAll so it doesn't have to repeat for every test case.